pypesto/visualize/profiles.py

```diff
@@ -120,6 +120,8 @@ def profiles_lowlevel(
         Lower bound.
     ub_full:
         Upper bound.
+    legend_text: List[str]
+        Label for line plots
```

Comment:
That docstring is already there in line 115

Reply:
yes, that was a merging error. thanks.